usageReports/es/es_route.go

```diff
@@ -0,0 +1,126 @@
+//   Copyright 2017 MSolution.IO
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+package es
+
+import (
+	"database/sql"
+	"net/http"
+	"time"
+
+	"github.com/trackit/trackit-server/db"
+	"github.com/trackit/trackit-server/routes"
+	"github.com/trackit/trackit-server/users"
+)
+
+// esQueryParams will store the parsed query params
+type (
+	esQueryParams struct {
+		accountList []string
+		indexList   []string
+		date        time.Time
+	}
+
+	esUnusedQueryParams struct {
+		accountList []string
+		indexList   []string
+		date        time.Time
+		count       int
+	}
+)
+
+var (
+	esQueryArgs = []routes.QueryArg{
+		routes.AwsAccountIdsOptionalQueryArg,
```

Comment:
You use `AwsAccountIdsOptionalQueryArg` instead of `AwsAccountsOptionalQueryArg`